.circleci/config.yml

```diff
@@ -40,12 +30,6 @@ jobs:
             path: doc/_build/html/
             destination: dev
 
-        # Store the data cache
-        - save_cache:
-            key: data-cache-0-{{ checksum "./mne_bids/__init__.py" }}
-            paths:
-              - ~/mne_data
```

Comment:
I don't think we should merge this right?

Reply:
Actually I did remove the caching here on purpose because we used outdated versions of some datasets, causing CI to fail … but I can revert this again and see how it goes

Reply:
You should be able to just bump the cache by incrementing the number: data-cache-0... --> data-cache-1...